packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -888,6 +892,11 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
     await Clipboard.setData(ClipboardData(text: data.plainText));
   }
 
+  void _selectAll() {
```

Comment:
Why create a new method since we already have selectAll public method.

It looks like the correct fix is to make that method to not hideToolbar ?

Reply:
thanks for comments. 
> Why create a new method since we already have selectAll public method.

nice question! I have optimized the method, base selectAll public method.

> It looks like the correct fix is to make that method to not hideToolbar ?

more accurately, In iOS or Android, after clicking `selectAll`, the handlers(startHandleLayerLink and endHandleLayerLink) are not hidden